lcviewernoqt/drawitems/lcvhatch.cpp

```diff
@@ -10,11 +15,72 @@ LCVHatch::LCVHatch(const lc::entity::Hatch_CSPtr& hatch) :
 }
 
 void LCVHatch::draw(LcPainter& painter, const LcDrawOptions &options, const lc::geo::Area& rect) const {
-    painter.circle(0, 0, 20);
-    painter.fill();
+    for(auto &x:_hatch->_loopList){
+        bool processingLine = false; //To make polyline from lines for hatch
+        for(auto &y:x->objList){
```

Comment:
Isn't that easier to use `asDrawable` from `DocumentCanvas` instead of casting them and calling the painter's functions ?

Reply:
i think that is not possible because we use fill method, if we change to asDrawable they will use stoke method,

Reply:
ok, i have an idea i have to change the fill method again.